x-pack/plugins/ml/public/application/explorer/anomaly_timeline.tsx

```diff
@@ -65,6 +68,8 @@ export const AnomalyTimeline: FC<AnomalyTimelineProps> = React.memo(
     // make sure dragSelect is only available if the mouse pointer is actually over a swimlane
     const disableDragSelectOnMouseLeave = useRef(true);
 
+    const canEditDashboards = capabilities.dashboard.createNew;
```

Comment:
When consuming capabilities from other applications, you can't assume that their capabilities property will exist:

```suggestion
    const canEditDashboards = capabilities.dashboard?.createNew ?? false;
```

Reply:
thanks @legrego ! what about other capabilities, e.g. `canEdit`? are they defined somewhere?

Reply:
The Dashboard app's capabilities are defined here, and they don't discriminate between create and edit. If users can create, then they can also edit: https://github.com/elastic/kibana/blob/b7e87c2c8b316ac6d567363f1eedd4322bdbf11d/x-pack/plugins/features/server/oss_features.ts#L162

It's also important to note that `createNew` will be set to `false` if the Dashboard application is simply hidden within the current space, even if the user would otherwise be authorized to create a dashboard. I'm not sure if that has any implications for your use case, but thought I should call it out.

Reply:
It is ok, the exact behavior I want to achieve. If the user currently in a space with a disabled Dashboard app or has no access - the control for initializing embeddable won't be rendered.